src/Bicep.Core/Semantics/Namespaces/AzNamespaceType.cs

```diff
@@ -21,7 +21,7 @@ public static class AzNamespaceType
             BicepProviderName: BuiltInName,
             ConfigurationType: null,
             ArmTemplateProviderName: "AzureResourceManager",
-            ArmTemplateProviderVersion: "1.0");
+            ArmTemplateProviderVersion: "1.0.0");
```

Comment:
Will this work with the Deployment Engine runtime / extensibility host? (same question for the other providers)

Reply:
Yes, it will work. The Deployment Engine and extensibility host don't check the provider versions. I've run an e2e test to confirm it.

Reply:
Thanks for checking!